File: output/CodeLlama/avatar/Java/Python/atcoder_AGC025_A.py
import sys

def sum(a):
    ans = 0
    while a >= 10:
        ans += a % 10
        a //= 10
    ans += a
    return ans


n = int(sys.stdin.readline().rstrip())
ans = sum(1) + sum(n - 1)
for A in range(1, n + 1):
    if sum(A) + sum(n - A) < ans:
        ans = sum(A) + sum(n - A)
print(ans)

File: output/ChatGPT/avatar/Java/C++/atcoder_AGC025_A.cpp
#include <iostream>
using namespace std;

int sum(int a) {
    int ans = 0;
    while (a >= 10) {
        ans += a % 10;
        a /= 10;
    }
    ans += a;
    return ans;
}

int main() {
    int N;
    cin >> N;
    int n = 0;
    if (N % 2 == 0) {
        n = N / 2;
    } else {
        n = (N + 1) / 2;
    }
    int ans = sum(1) + sum(N - 1);
    for (int A = 1; A <= n; A++) {
        if (sum(A) + sum(N - A) < ans) {
            ans = sum(A) + sum(N - A);
        }
    }
    cout << ans << endl;
}